vpn/src/main/java/com/duckduckgo/mobile/android/vpn/network/connection/ConnectivityChangeReceiver.kt

```diff
@@ -47,6 +49,16 @@ class ConnectivityChangeReceiver @Inject constructor(
             val networkType = intent.getIntExtra(ConnectivityManager.EXTRA_NETWORK_TYPE, ConnectivityManager.TYPE_DUMMY)
             if (networkType == ConnectivityManager.TYPE_VPN) return
 
+            val oldNetworkType = vpnPreferences.activeNetworkType
+
+            if (oldNetworkType == networkType.toString()) {
```

Comment:
should we use `networkType.lowercase()` instead? just in case some OEMs don't respect upppercase / lowercase?